src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -69,6 +70,13 @@ private ConversationStore(PersistentStorageAgent persistentStorageAgent) {
     return conversations;
   }
 
+    /**
+     * Gets all public conversations known to the application
+     */
+    public List<Conversation> getAllPublicConversations() {
+        return conversations.stream().filter(conversation -> !conversation.isPrivate()).collect(Collectors.toList());
```

Comment:
Functional programming for the win!! :D So much cleaner than a for loop

Reply:
Efficient!